modules/xfem/src/userobjects/CrackMeshCut3DUserObject.C

```diff
@@ -765,28 +822,64 @@ MeshCut3DUserObject::growFront()
       Point dir = _active_direction[i][j];
 
       Point x;
-      for (unsigned int k = 0; k < LIBMESH_DIM; ++k)
-        x(k) = this_point(k) + dir(k) * _size_control;
+
+      if (_growth_speed_method == "function")
+        for (unsigned int k = 0; k < LIBMESH_DIM; ++k)
```

Comment:
We kind of have a convention of using LIBMESH_DIM rather than 3, but I think it's better to just set it to 3.  If you happened to compile with LIBMESH_DIM set to anything else, this would all break -- it's a 3D mesh cutter, after all!

Reply:
Done